sirepo/server.py

```diff
@@ -697,6 +689,18 @@ def all_uids():
     return oauth.all_uids(app)
 
 
+@app.after_request
+def app_after_request(response):
+    if re.search(r'^(2|3)', str(response.status_code)):
+        flask.session[_SESSION_KEY_VALID] = 1
```

Comment:
You don't want to change the cookie if is already is valid. I think it's a bit confusing to use the word "valid", but rather something like "cookie sentinel". It's only used for the case when we don't have this value so the logic looks like:

flask.session.setdefault('_SESSION_KEY_COOKIE_SENTINEL', 1)

The sentinel can just be "ok" in the cookie itself.

Reply:
The app.secret_key is using the existing config - cfg.beaker_session.secret. I re-used the beaker_session key and secret for the session cookie. That way existing config doesn't need to change.